react-stopwatch-component/dist/styles.css

```diff
@@ -0,0 +1,27 @@
+* {
+  font-family: sans-serif, "Times New Roman", Times, serif;
+  font-size: 17px;
+}
+
+.container {
+  /* border: 1px solid red; */
+  max-width: 800px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.circle {
+  background-color: rgb(249 249 249);
+  height: 150px;
+  width: 150px;
+  border-radius: 100px;
+  border-color: rgb(212 209 209 / 25%);
+  box-shadow: 5px 2px 2px rgb(212 209 209);
+}
+
+.fa-play,
+.fa-pause {
+  margin-top: 30px;
+  margin-left: 8.8px;
+  color: rgb(80 75 75);
+}
```

Comment:
You shouldn't directly add custom CSS to existing classes from CSS libraries. It is better to make your own class and use that on the elements that you want it applied to.